lib/Log/ApplicationLogger.php

```diff
@@ -158,7 +158,7 @@ public function log($level, $message, array $context = [])
                 $context['fileObject'] = str_replace(PIMCORE_PROJECT_ROOT, '', $context['fileObject']->getFilename());
             }
 
-            file_put_contents(PHP_EOL.$context['fileObject'], $message, FILE_APPEND);
+            file_put_contents(PHP_EOL.$context['fileObject'], PHP_EOL.'['.$level.'] '.$message, FILE_APPEND);
```

Comment:
```suggestion
            file_put_contents($context['fileObject'], PHP_EOL.'['.$level.'] '.$message, FILE_APPEND);
```

Reply:
Thanks, fixed this. I should not edit files manually after testing ;-)